server/incentiveService.js

```diff
@@ -5,9 +5,8 @@ const sendCheckoutToIncentive = order => (
     const incentive = {
       userid: order.userid,
       address: order.address,
-      shipping: order.shipping,
+      cart: order.cart,
```

Comment:
I'll be handling this so you wont need this part anymore

Reply:
Sounds good. I'll comment it out and leave it there in case we change our systems architecture